Web/Components/Pages/Settings.razor.css

```
@@ -0,0 +1,3 @@
+::deep .settings-card {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)